src/Lachain.Core/RPC/HTTP/Web3/TransactionServiceWeb3.cs

```diff
@@ -483,7 +483,7 @@ public string Call(JObject opts, string? blockId)
         
                 return transferInvRes.Status == ExecutionStatus.Ok
                     ? (gasUsed + transferInvRes.GasUsed).ToHex()
-                    : Web3DataFormatUtils.Web3Number(gasUsed);
+                    : throw new Exception("Error in funds transfer");
```

Comment:
Maybe we can use rpc exception as we know the reason of error. It could be cleaner.

Reply:
ok,  I'll add it